charts/kube-prometheus-stack/values.yaml

```
@@ -876,6 +876,14 @@ alertmanager:
     ## be considered available. Defaults to 0 (pod will be considered available as soon as it is ready).
     minReadySeconds: 0
 
+    ## Additional configuration which is not covered by the properties above. (passed through tpl)
+    additionalConfig: {}
+
+    ## Additional configuration which is not covered by the properties above.
+    ## Useful, if you need advanced templating inside alertmanagerSpec.
+    ## Otherwise, use alertmanager.alertmanagerSpec.additionalConfig (passed through tpl)
+    additionalConfigString: {}
```

Thanks @jkroepke, this causes the following output on the lint job:

[1905](https://github.com/prometheus-community/helm-charts/actions/runs/6584932364/job/17890388114?pr=3920#step:9:1906) coalesce.go:220: warning: cannot overwrite table with non table for kube-prometheus-stack.alertmanager.alertmanagerSpec.additionalConfigString (map[])
[1906](https://github.com/prometheus-community/helm-charts/actions/runs/6584932364/job/17890388114?pr=3920#step:9:1907) coalesce.go:220: warning: cannot overwrite table with non table for kube-prometheus-stack.alertmanager.alertmanagerSpec.additionalConfigString (map[])

I think you want this:

```suggestion
    additionalConfigString: ""
```